client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/packages-step/package-select.js

```diff
@@ -109,7 +109,7 @@ const PackageSelect = props => {
 			</div>
 			{ 0 === totalPackagesCount ? (
 				<div className="packages-step__no-packages">
-					<Gridicon icon="product" size="20" />
+					<Gridicon icon="product" size={18} />
```

Comment:
`20` is not allowed in the Girdicons `size` settings.
>  error  Gridicon size should be one of recommended sizes: 12, 18, 24, 36, 48, 54, 72
Tried using both `18` and `24`. 
`18` looks like 
![image](https://user-images.githubusercontent.com/572862/88855526-4167ea80-d1b0-11ea-96ff-7bdf06f8b246.png)

`24` looks like 
![image](https://user-images.githubusercontent.com/572862/88855544-475dcb80-d1b0-11ea-86d5-b98f7fd1e973.png)

18 seems to be closer to the original 20. So i am using 18 here.